.github/workflows/l10n_extract.yaml

```diff
@@ -41,10 +41,12 @@ jobs:
           yarn workspace fxa-settings grunt merge-ftl
           yarn workspace fxa-auth-server grunt merge-ftl
           yarn workspace fxa-react grunt merge-ftl
+          yarn grunt --gruntfile='apps/payments/next/Gruntfile.js' merge-ftl
```

Comment:
Testing this locally, the script fails when I attempt to run the command locally in terminal with `yarn grunt --gruntfile='apps/payments/next/Gruntfile.js' merge-ftl` unless I install things via `yarn install` with the error: `Usage Error: Couldn't find a script named "grunt".`

This is also failed when I attempted to run it from the fork: https://github.com/bcolsson/fxa-content-server-l10n/actions/runs/12658750573/job/35276377203